src/comparable.ml

```diff
@@ -0,0 +1,13 @@
+module type S = sig
+  type t
+
+  include Equatable.S with type t := t
+
+  val compare:t Comparator.t
+end
+
+module Default(T: sig type t end) : S = struct
```

Comment:
Can I highly recommend not including structural equality and comparison in bs-base. In practice it's very slow, and almost always the wrong choice.